.github/workflows/testing.yml

```diff
@@ -0,0 +1,19 @@
+name: "Testing"
```

Comment:
I'd call it `"Test"` in imperative. Same for the workflow file.

Reply:
Changed 👍